test/test_image_checker.py

```diff
@@ -216,6 +216,28 @@ def test_is_use_default_domain(self, mock):
             self.assertRaises(RuntimeError,
                               mock_image_checker._get_normalized_image_info)
 
+    @patch.object(ImageChecker, "__init__")
+    def test_get_registry_from_image_uri(self, mock):
+        mock.return_value = None
+        mock_image_checker = ImageChecker({}, {})
+
+        test_cases = [{
+            "image_uri": "registry.domain:5000/user-name/repo-0:tag-0.version",
+            "expect_registry": "https://registry.domain:5000/v2/"
```

Comment:
why expect https

Reply:
Will add comments for image checker. It currently only support registry which enabled `https`. For http registry, image checker will exit 0, do not block following steps